service/src/main.rs

```diff
@@ -478,7 +495,8 @@ fn execute_update_market<E: EnclaveBase + TeeracleApi>(
 	enclave: &E,
 ) {
 	// Get market data for usd (hardcoded)
-	let updated_extrinsic = match enclave.update_market_data_xt(node_api.genesis_hash, "usd") {
+	let updated_extrinsic = match enclave.update_market_data_xt(node_api.genesis_hash, "DOT", "USD")
```

Comment:
This is still intentionally hardcoded?

Reply:
Yes. It's from the old demo https://github.com/integritee-network/worker/issues/292
But I will change the crypto currency to TEER